javascript/javascript1/week1/housey_pricey.js

```diff
@@ -0,0 +1,30 @@
+//Peter`s case:
+let width = 8;
+let depth = 10;
+let heght = 10;
```

Comment:
height - spelling correctly is crucial in large distributed systems in order to collaborate and understand your teammates 🥇.

Reply:
Got it! Thank you!